src/pages/LogIn/components/LoginForm.tsx

```diff
@@ -51,6 +54,7 @@ class LoginFormComponent extends React.Component<AllProps, IState> {
       );
     } catch (error) {
       this.setState({
+        errorMessage:(()=>error.response?'Invalid Credentials':'There was a problem connecting to the server!')(),
```

Comment:
I suppose [`@fnndsc/chrisapi`](https://github.com/FNNDSC/fnndsc/tree/master/js/chrisAPI) itself does not have the best exception objects. Maybe a PR there first would help?

But still you could be doing a `error.response.status === 200` or some other strong assertion.

```json
{
  "columnNumber": 12,​
  "fileName": "http://localhost:3000/static/js/vendors~main.chunk.js",
​  "lineNumber": 14154,
​  "message": "Bad server response!",
  "name": "o",​
  "request": XMLHttpRequest { readyState: 4, timeout: 30000, withCredentials: false, ... },
​  "response": { 
    "data": "Bad server response!",
    "status": 404,
    "statusText": "Not Found",
    ...
  },
  "stack": "a super long string..."
}
```